### DISCUSSION TITLE
Show commenter name

### DISCUSSION THREAD

**Discussion by pluma9** [ankitpokhrel/jira-cli#612] [Ideas]
Currently, commenter names are not displayed. Commenter name is too important to be omitted.

**Comment by ankitpokhrel** [comment]
Hi @pluma9, This seem to be a bug. The name [is not omitted ](https://github.com/ankitpokhrel/jira-cli/blob/main/internal/view/issue.go#L405), but it seems that `Name` may not always be present. Will have to check if `DisplayName` is available before falling back to the `Name` field. Thank you for reporting!

**Comment by ankitpokhrel** [comment]
This should be fixed with #618 and is available in the new version [v1.4.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.4.0)
